test/draftertest.h

```diff
@@ -12,20 +12,25 @@
 #include "SerializeResult.h"
 #include "sosJSON.h"
 
+
+#define TEST_DRAFTER(description, category, name, tag,  wrapper, options, mustBeOk) TEST_CASE(description " " category " " name, "[" tag "][" category "]") { \
+    REQUIRE(FixtureHelper::handleResultJSON(wrapper, "test/fixtures/" category "/" name, options, mustBeOk)); \
+}
```

Comment:
Okay, we can do that later then. Please fix the other comment and I can merge this.